src/connectivity/rest/rest_controller.py

```diff
@@ -5,8 +5,8 @@
 from flask_restful import Resource, Api
 
 from configuration import Configuration as configuration
-from src.common_utils.database.collection_utils import parse_documents
-from src.common_utils.database.database_service import DatabaseProxy
+from src.database.database.collection_utils import parse_documents
+from src.database.database.database_service import DatabaseProxy
```

Comment:
it will be better to not use the same name for following folders (as here database.database)